services/gateway/tests/functional/registerHttpApi.test.js

```diff
@@ -0,0 +1,582 @@
+		expect(response).toEqual({
+			data: [
+				{
+					chainID: '03000000',
+					chainName: 'Lisk',
+					tokenID: '0300000000000000',
+					tokenName: 'Lisk',
+					networkType: 'alphanet',
+					description: 'Default token for the entire Lisk ecosystem',
+					logo: {
+						png: 'https://lisk-qa.ams3.digitaloceanspaces.com/Artboard%201%20copy%2019.png',
+						svg: 'https://lisk-qa.ams3.digitaloceanspaces.com/Logo-20.svg',
+					},
+					symbol: 'LSK',
+					displayDenom: 'lsk',
+					baseDenom: 'beddows',
+					denomUnits: [
+						{
+							denom: 'beddows',
+							decimals: 0,
+							aliases: [
+								'Beddows',
+							],
+						},
+						{
+							denom: 'lsk',
+							decimals: 8,
+							aliases: [
+								'Lisk',
+							],
+						},
+					],
+					customNumber: 123,
+				},
+				{
+					chainID: '04000000',
+					chainName: 'Lisk',
+					tokenID: '0400000000000000',
+					tokenName: 'Lisk',
+					networkType: 'devnet',
+					description: 'Default token for the entire Lisk ecosystem',
+					logo: {
+						png: 'https://lisk-qa.ams3.digitaloceanspaces.com/Artboard%201%20copy%2019.png',
+						svg: 'https://lisk-qa.ams3.digitaloceanspaces.com/Logo-20.svg',
+					},
+					symbol: 'LSK',
+					displayDenom: 'lsk',
+					baseDenom: 'beddows',
+					denomUnits: [
+						{
+							denom: 'beddows',
+							decimals: 0,
+							aliases: [
+								'Beddows',
+							],
+						},
+						{
+							denom: 'lsk',
+							decimals: 8,
+							aliases: [
+								'Lisk',
+							],
+						},
+					],
+					customNumber: 456,
+				},
+			],
+			meta: {
+				count: 2,
+				offset: 0,
+				total: 5,
+			},
+		});
+	});
+
+	it('should return data when called with null methodDef', async () => {
+		const response = await transformResponse(null, data);
+		expect(response).toEqual(data);
+	});
+
+	it('should throw when called with null data', async () => {
+		expect(() => transformResponse(methodDef, null)).rejects.toThrow();
+	});
+
+	it('should return data when called with undefined methodDef', async () => {
+		const response = await transformResponse(undefined, data);
+		expect(response).toEqual(data);
+	});
+
+	it('should throw when called with undefined data', async () => {
+		expect(() => transformResponse(methodDef, undefined)).rejects.toThrow();
+	});
+
+	it('should return null when called with null methodDef and data', async () => {
+		const response = await transformResponse(null, null);
+		expect(response).toEqual(null);
+	});
+
+	it('should return undefined null when called with undefined methodDef and data', async () => {
+		const response = await transformResponse(undefined, undefined);
+		expect(response).toEqual(undefined);
+	});
+});
+
+describe('Test transformResponse method', () => {
+	const apiNames = ['http-version3', 'http-exports'];
+	const config = {
+		whitelist: [],
+		aliases: {},
+	};
+	const registeredModuleNames = ['fee', 'interoperability', 'legacy', 'pos', 'random', 'token', 'validators'];
+	const expectedResponse = {
```

Comment:
I think its better to move all test data including expected responses to the `constants` directory.

Reply:
Applied.